### DISCUSSION TITLE
Will this work?

### DISCUSSION THREAD

**Discussion by lectrical** [dangowrt/owrt-ubi-installer#157] [General]
If I updated the openwrt version and the pgp key in the script and it builds successful does that mean it will work and not brick my router?

https://github.com/lectrical/owrt-ubi-installer/releases/tag/v1.0.5

tbh i was just playing around, i have not tried to install anything yet, but if this would work, i might just do that?

`openwrt-23.05.0-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb`

**Comment by dangowrt** [comment]
99% it will work, but to be safe you can wait a couple of days until I had a chance to test the resulting images on real hardware (with jtag already attached to ease recovery in case something goes wrong)

**Reply by lectrical** [reply]
edit: i saw the script downloads them so it makes sense they are the same

I did it the readme way and updated after, then i noticed mine was the exact same size and once i understood the process i bit better i figured i could checksum the sysupgrade files?

These piped checksum commands give the same result, so i figure my build is identical to theirs?

```
curl -sLo- https://github.com/lectrical/owrt-ubi-installer/releases/download/v1.0.5/openwrt-23.05.0-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb | sha1sum
curl -sLo- https://downloads.openwrt.org/releases/23.05.0/targets/mediatek/mt7622/openwrt-23.05.0-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb | sha1sum
```

```
7d91ab6e593472ba202a363c4c09ddf3d4d48e55  -
```

So I would like to know if it passed testing before using it but i'm on 23.05.0 now anyway,
